src/cpp/rtps/participant/RTPSParticipant.cpp

```diff
@@ -145,7 +145,7 @@ WLP* RTPSParticipant::wlp() const
     return mp_impl->wlp();
 }
 
-fastdds::dds::builtin::TypeLookupManager* RTPSParticipant::typelookup_manager() const
+fastdds::dds::builtin::TypeLookupManager& RTPSParticipant::typelookup_manager() const
 {
     return mp_impl->typelookup_manager();
 }
```

Comment:
This API should be removed. The TypeLookup Manager should not be exposed publicly.

Reply:
Removed in [9cf4645](https://github.com/eProsima/Fast-DDS/pull/4126/commits/9cf46451cbc0cc26f54f25c7979ada4904c22109)